src/objects/Course_Offering__c.object

```diff
@@ -20,10 +20,6 @@
         <actionName>Edit</actionName>
         <type>Default</type>
     </actionOverrides>
-    <actionOverrides>
```

Comment:
I noticed this & some of the other changes in this file on the other existing objects too - I think it's a result of moving to v45.0. I wonder if we should sync the other objects too?

Reply:
We should try to research this. I have been seeing this as well.

Reply:
For this one specifically, I think it's safe to do. I don't see "follow" in the list of action overrides so I'm assuming it was removed at some point: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/actionoverride.htm